lib/util/mqtt/app.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:io';
+
+import 'package:mqtt_client/mqtt_client.dart';
+import 'package:mqtt_client/mqtt_server_client.dart';
+
+MqttClient setup(String serverAddress, String uniqueID, int port) {
+  return MqttServerClient.withPort(serverAddress, uniqueID, port)
```

Comment:
To follow the camelCase

```suggestion
  return MqttServerClient.withPort(serverAddress, uniqueId, port)
```